pkgs/by-name/aq/aquamarine/package.nix

```diff
@@ -51,10 +57,9 @@ stdenv.mkDerivation (finalAttrs: {
     udev
     wayland
     wayland-protocols
+    wayland-scanner
   ];
 
-  strictDeps = true;
```

Comment:
Why drop that? Would a move of wayland-scanner to nativeBuildInputs be enough?

Reply:
Dropping `strictDeps` happened by accident. I re-introduced it and it still builds.